.github/workflows/release-gh-pages.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          token: ${{ secrets.PAT_GITHUB }}
+          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Maybe we could add a comment here to explain why we have this `GH_PERSONAL_ACCESS_TOKEN` instead of the normal `GITHUB_TOKEN` that is used below?

Reply:
yeap. done. thanks